client/src/components/ShareDraftContent/ShareDraftContent.js

```diff
@@ -121,6 +121,8 @@ class ShareDraftContent extends Component {
           rel="noopener noreferrer"
         >
           {i18n._t('ShareDraftContent.LEARN_MORE', 'Learn more')}
+          <span className="share-draft-content__learn-more__icon font-icon-external-link" aria-hidden="true" />
+          <span className="visually-hidden">({i18n._t('ShareDraftContent.EXTERNAL_LINK', 'external link')})</span>
```

Comment:
This icon isn't purely decorative, so we need to give screen reader users some context in the form of localised visually-hidden text.